libraries/ESP8266WiFiMesh/src/ESP8266WiFiMesh.cpp

```diff
@@ -169,9 +169,9 @@ void ESP8266WiFiMesh::deactivateAP()
 void ESP8266WiFiMesh::restartAP()
 {
   deactivateAP();
-  yield();
+  optimistic_yield(10000);
```

Comment:
I'm concerned about the change here.  They're obviously worried that deactivateAP() can be very long and that activateAP() can also be long.

If deactivateAP() takes 9ms, the 1st optimistic_yield() won't fire.  Then, if activateAP() takes 9ms, you will have 18ms between yields() here, whereas before you'd be only 9ms between them.
Before, they'd get 2 yields no matter what, but here, if activateAP() is long, you could

Reply:
@earlephilhower Of course you're making up the numbers to prove your case, but your on to something, and it's indeed a good catch that NOT yielding here make keep the firmware from running to actually carry the op to completion. I don't know, also optimistic_yield is best for loops, and I too don't think this will be called in a tight loop. So I'm reverting these two changes.